README.md

```diff
@@ -63,139 +68,219 @@ Use `#pragma mark -` to categorize methods in functional groupings and protocol/
+* `NSAttributedString *titleAttributedString`: A title, already formatted for display. _`AttributedString` hints that this value is not just a vanilla title, and adding it could be a reasonable choice depending on context._
+* `NSDate *now`: _No further clarification is needed._
+* `NSDate *lastModifiedDate`: Simply `lastModified` can be ambiguous; depending on context, one could reasonably assume it is one of a few different types.
+* `NSURL *URL` vs. `NSString *URLString`: In situations when a value can reasonably be represented by different classes, it is often useful to disambiguate in the variable’s name.
+* `NSString *releaseDateString`: Another example where a value could be represented by another class, and the name can help disambiguate.
+
+Single letter variable names should be avoided except as simple counter variables in loops.
+
+Asterisks indicating a type is a pointer should be “attached to” the variable name. **For example,** `NSString *text` **not** `NSString* text` or `NSString * text`, except in the case of constants (`NSString * const NYTConstantString`).
+
+### Constants
+
+Constants are preferred over in-line string literals or numbers, as they allow for easy reproduction of commonly used variables and can be quickly changed without the need for find and replace. Constants should be declared as `static` constants and not `#define`s unless explicitly being used as a macro.
 
 **For example:**
+
 ```objc
-view.backgroundColor = [UIColor orangeColor];
-[UIApplication sharedApplication].delegate;
+static NSString * const KCPAboutViewControllerCompanyName = @"Kinetic Cafe Inc.";
+
+static const CGFloat KCPImageThumbnailHeight = 50.0;
 ```
 
 **Not:**
+
 ```objc
-[view setBackgroundColor:[UIColor orangeColor]];
-UIApplication.sharedApplication.delegate;
+#define CompanyName @"Kinetic Cafe Inc."
+
+#define thumbnailHeight 2
 ```
 
-## Spacing
+### Prefix
 
-* Indent using 4 spaces. Never indent with tabs. Be sure to set this preference in Xcode.
-* Method braces and other braces (`if`/`else`/`switch`/`while` etc.) always open on the same line as the statement but close on a new line.
+* A three letter prefix (e.g., `KCP`) should always be used for class names and constants. Constants should be camel-case with all words capitalized and prefixed by the related class name for clarity. A two letter prefix (e.g., `NS`) is [reserved for use by Apple](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/DefiningClasses/DefiningClasses.html#//apple_ref/doc/uid/TP40011210-CH3-SW12).
+* Constant name should follow the format of `<filename>constantName`
 
 **For example:**
+
 ```objc
-if (user.isHappy) {
-    // Do something
-}
-else {
-    // Do something else
-}
+static const NSTimeInterval KCPNavigationViewControllerNavigationFadeAnimationDuration = 0.3;
```

Comment:
Yeah, we talked about it yesterday. We decided to drop the "k" because we already have the prefix and descriptive long name.